service/resource/configmap/service.go

```diff
@@ -231,6 +231,12 @@ func (s *Service) GetDeleteState(obj, currentState, desiredState interface{}) (i
 	return deleteState, nil
 }
 
+// GetUpdateState currently returns nil values because this is a simple resource
+// not concerned with being updated, just fulfilling the resource interface
```

Comment:
❤️  The GoDoc comments but nitpick they should end with a full stop.

Reply:
Sure! done :)